database/abusescanner.go

```diff
@@ -302,6 +310,17 @@ func (db *AbuseScannerDB) FindUnparsed() ([]AbuseEmail, error) {
 	return emails, nil
 }
 
+// Purge removes all documents from the emails and locks collection
+func (db *AbuseScannerDB) Purge(ctx context.Context) error {
```

Comment:
I consider it a method on the `AbuseScannerDB`. Moving it to a test package would make things awkward because we'd have to expose the collections and the tests calling it live in a separate package.

Reply:
How much effort would it be to just connect to the db from the test and expose the name of the collections for the test to prune it? 
I agree that having this exposed just for testing isn't great.